package.json

```diff
@@ -6,9 +6,12 @@
   "scripts": {
     "clean": "rm -rf ./dist",
     "dev": "webpack serve --config webpack.dev.js",
-    "build": "webpack --config webpack.prod.js",
+    "dev:vhs": "webpack serve --config webpack.dev.js --env vhs",
+    "build": "npm run build:hlsjs && npm run build:vhs",
+    "build:hlsjs": "webpack --config webpack.prod.js",
+    "build:vhs": "webpack --config webpack.prod.js --env vhs",
     "test": "npm run test:integrations && npm run test:e2e",
-    "test:integrations": "jest -c ./test/jest.config.js",
+    "test:integrations": "jest -c ./test/jest.config.js --passWithNoTests",
```

Comment:
Since `mux-protocol.js` was removed, there are no longer any integration tests, so, this should pass without any tests.